manifests/init.pp

```diff
@@ -70,8 +70,13 @@
   }
 
   if $install_package == true {
-    package { 'ca-certificates':
-      ensure => $package_ensure,
+    if $package_ensure == present {
```

Comment:
Could you add to this so it also matches on `$package_ensure == installed` since they're considered equivalent?